jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile

```diff
@@ -0,0 +1,162 @@
+            }
+            post {
+               always {
+                   sh "cd ${TESTDIR}; make clean-jenkins-precheck || true"
+               }
+               unsuccessful {
+                   script {
+                       if  (!currentBuild.displayName.contains('fail')) {
+                           currentBuild.displayName = 'precheck fail\n' + currentBuild.displayName
+                       }
+                   }
+               }
+            }
+        }
+        stage('Preload vagrant boxes') {
+            steps {
+                sh '/usr/local/bin/add_vagrant_box ${WORKSPACE}/${PROJ_PATH}/vagrant_box_defaults.rb'
+            }
+            post {
+                unsuccessful {
+                    script {
+                        if  (!currentBuild.displayName.contains('fail')) {
+                            currentBuild.displayName = 'preload vagrant boxes fail' + currentBuild.displayName
+                        }
+                    }
+                }
+            }
+        }
+        stage ("Copy code and boot vms"){
+            options {
+                timeout(time: 30, unit: 'MINUTES')
+            }
+
+            environment {
+            CONTAINER_RUNTIME=setIfLabel("area/containerd", "containerd", "docker")
+                GOPATH="${WORKSPACE}/${TESTED_SUITE}-gopath"
+                TESTDIR="${GOPATH}/${PROJ_PATH}/test"
+                KUBECONFIG="vagrant-kubeconfig"
+            }
+            steps {
+                sh 'mkdir -p ${GOPATH}/src/github.com/cilium'
+                sh 'cp -a ${WORKSPACE}/${PROJ_PATH} ${GOPATH}/${PROJ_PATH}'
+                retry(3) {
+                    timeout(time: 30, unit: 'MINUTES'){
+                        dir("${TESTDIR}") {
+                            sh 'vagrant destroy runtime --force'
+                            sh 'KERNEL=$(python get-gh-comment-info.py "${ghprbCommentBody}" --retrieve=version | sed "s/^$/${DEFAULT_KERNEL}/") vagrant up runtime --provision'
+                        }
+                    }
+                }
+            }
+            post {
+                unsuccessful {
+                    script {
+                        if  (!currentBuild.displayName.contains('fail')) {
+                            currentBuild.displayName = 'runtime vm provisioning fail\n' + currentBuild.displayName
+                        }
+                    }
+                }
+            }
+        }
+        stage ("BDD-Test-PR"){
+            options {
+                timeout(time: 180, unit: 'MINUTES')
+            }
+            environment {
+                GOPATH="${WORKSPACE}/${TESTED_SUITE}-gopath"
+                TESTDIR="${GOPATH}/${PROJ_PATH}/test"
+            }
+            steps {
+                sh 'cd ${TESTDIR}; ginkgo --focus="$(python get-gh-comment-info.py "${ghprbCommentBody}" | sed "s/^$/Runtime.*/" | sed "s/K8s.*/NoTests/")" -v --failFast=${FAILFAST} -- -cilium.provision=false -cilium.timeout=${GINKGO_TIMEOUT}'
```

Comment:
Is the python script new?

Reply:
it's used in kernel-specific jenkinsfile